test/test_manifest.json

```diff
@@ -1451,6 +1451,13 @@
        "type": "eq",
        "nativeImageDecoderSupport": "none"
     },
+    {  "id": "issue10529",
+       "file": "pdfs/issue10529.pdf",
+       "md5": "1a4d404a137c610ff0c747cbea3b8666",
+       "rounds": 1,
+	   "link": false,
```

Comment:
Please, no tabs anywhere in the code-base.

Also, please follow https://github.com/mozilla/pdf.js/wiki/Squashing-Commits

Reply:
I didn't notice a tab, thanks.